python/lsst/afw/math/_backgroundList.py

```diff
@@ -79,6 +79,9 @@ def __getitem__(self, *args):
     def __len__(self, *args):
         return self._backgrounds.__len__(*args)
 
+    def __bool__(self):
+        return bool(self._backgrounds)
+
```

Comment:
I'm actually a little surprised this is necessary - I thought defining `__len__` would do it.  But I see you have a test.

Reply:
You're right. I can skip the `__bool__` but would like to keep the tests.